ledfx/api/virtual_tools.py

```diff
@@ -48,11 +49,11 @@ async def put(self, request: web.Request) -> web.Response:
                     virtual.force_frame(parse_color(validate_color(color)))
 
         if tool == "oneshot":
-            color = data.get("color", "white")
-
+            color = parse_color(validate_color(data.get("color", "white")))
             ramp = data.get("ramp", 0)
             hold = data.get("hold", 0)
             fade = data.get("fade", 0)
+            brightness = data.get("brightness", 0)
```

Comment:
Should brightness default to 1, not zero? If old api call has ko brightness, then I am assuming we want full?

Woll peruse more code, but still phone only.